pkg/resources/deployments/deployment.go

```diff
@@ -107,6 +107,27 @@ func NewDeploymentForCR(m *v1alpha1.Interconnect) *appsv1.Deployment {
 			})
 		}
 	}
+	if len(m.Spec. Users) > 0 {
```

Comment:
typo, m.Spec.Users

Reply:
odd... how did that even compile? Unless I accidentally hit space on final commit...